### ISSUE TITLE
Make Sections Collapsable (UI)

### ISSUE THREAD

**Issue by Kozzren** [leukbaars/DreamUV#41]
For a nice qol touch, I think have each section be collapsable would be nice, instead of it be scrollable etc

**Comment by leukbaars** [comment]
Agree! Been thinking the same now that I've added a bunch of tools. I'll add this next as its easy
